components/automate-chef-io/content/docs/data-collection.md

```diff
@@ -21,7 +21,7 @@ There are two steps to getting data collection running in Chef Automate:
 
 1. You must first have an API token. You have two options:
 
-  * [Create a new API Token]({{< relref "api-tokens.md#creating-api-tokens" >}}).
+  * [Create a new API Token]({{< relref "api-tokens.md#creating-api-tokens" >}}) and add it to the Ingest policy.
   * Or you can [use your existing data collector token]({{< relref "#existing-data-collector-token-setup" >}}) if you are migrating from Chef Automate 1.
 
 1. Once you have an API token, you can either:
```

Comment:
NTS: This numbering format needs to be fixed. Shows two `1.` in a row.

Reply:
@susanev I'm happy to fix this, but I need to leave myself a note so I don't forget! :)

Reply:
I'll fix this in a separate PR!